www/js/survey/multilabel/MultiLabelButtonGroup.tsx

```diff
@@ -151,16 +191,47 @@ const MultilabelButtonGroup = ({ trip, buttonsInline = false }) => {
             </Dialog.Title>
             <Dialog.Content style={{ maxHeight: windowHeight / 2, paddingBottom: 0 }}>
               <ScrollView style={{ paddingBottom: 24 }}>
+                <Text style={{ fontSize: 12, color: '#333333', paddingVertical: 4 }}>
+                  Default Mode
+                </Text>
                 <RadioButton.Group onValueChange={(val) => onChooseLabel(val)} value={chosenLabel}>
-                  {labelOptions?.[modalVisibleFor]?.map((o, i) => (
+                  {labelOptions?.[modalVisibleFor]?.map((o, i) => {
+                    // This code is designed to order the modes as follows: default modes -> customized modes -> other
+                    if (o.value === 'other') return;
+                    return (
+                      // @ts-ignore
+                      <RadioButton.Item
+                        key={i}
+                        label={t(o.text)}
+                        value={o.value}
+                        style={{ paddingVertical: 2 }}
+                      />
+                    );
+                  })}
+                  {customModes.length > 0 && (
+                    <>
+                      <Divider style={{ marginVertical: 10 }} />
+                      <Text style={{ fontSize: 12, color: '#333333', paddingVertical: 4 }}>
+                        Custom Mode
```

Comment:
This will need to be internationalized instead of using direct English text
Add a new translation entry to `en.json` and use the `t` function here.